src/vconsole/vconsole-setup.c

```diff
@@ -282,6 +285,7 @@ int main(int argc, char **argv) {
         utf8 = is_locale_utf8();
 
         r = parse_env_file("/etc/vconsole.conf", NEWLINE,
+                           "VC_MAX", &vc_max_s,
```

Comment:
why is this configurable? What's the precise effect of this new setting?

This needs documentation in the man pages for vconsole.conf as well as for the kernel cmdline.